monkey/infection_monkey/monkey.py

```diff
@@ -234,8 +236,8 @@ def start(self):
                     if not self._keep_running:
                         break
 
-                ransomware_attempts = RansomewarePayload(WormConfiguration.ransomware).run_payload()
-                RansomwareTelem(ransomware_attempts).send()
+                telemetry_messenger = TelemetryMessengerWrapper()
```

Comment:
Why instantiate this here instead of inside `RansomwarePayload`?

Reply:
(If my assumption in point 2 of my main comment is correct.)

Reply:
See response to https://github.com/guardicore/monkey/pull/1259/files#r658519175